sor/src/main/java/com/bazaarvoice/emodb/sor/db/astyanax/RecordImpl.java

```diff
@@ -14,13 +15,13 @@
 
 class RecordImpl implements Record {
     private final Key _key;
-    private Iterator<Map.Entry<UUID, Compaction>> _passOneIterator;
-    private Iterator<Map.Entry<UUID, Change>> _passTwoIterator;
+    private Iterator<Map.Entry<DeltaClusteringKey, Compaction>> _passOneIterator;
+    private Iterator<Map.Entry<DeltaClusteringKey, Change>> _passTwoIterator;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "_passTwoIterator" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS00116)